src/lib.rs

```diff
@@ -3,3 +3,5 @@ pub mod block_engine;
 pub mod error;
 pub mod proto;
 pub mod searcher;
+mod client;
+mod pem;
```

Comment:
```suggestion
pub mod auth;
pub mod block_engine;
pub mod client;
pub mod error;
mod pem;
pub mod proto;
pub mod searcher;
```
`client` module must be public